content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/delivery_three_steps.rst

```diff
@@ -52,64 +52,46 @@ quotation form. On the blank quotation form, select a :guilabel:`Customer`, add
 :guilabel:`Product`, and click :guilabel:`Confirm`.
 
 A :guilabel:`Delivery` smart button appears in the top right of the quotation form. Clicking it
-shows the picking order, packing order, and the delivery order associated with the sales order.
+opens the picking order to move the ordered product from `WH/Stock` to `WH/Packing Zone`.
 
-.. image:: delivery_three_steps/three-step-delivery-so.png
+.. image:: delivery_three_steps/delivery-three-steps-smart-button.png
    :align: center
    :alt: After confirming the sales order, the Delivery smart button appears showing three items
          associated with it.
 
 Process a picking
 -----------------
 
-The picking, packing, and delivery orders will be created once the sales order is confirmed.  To
-view these transfers, go to :menuselection:`Inventory --> Operations --> Transfers`.
+The picking order will be created once the sales order is confirmed. To view the picking, navigate
+to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
+:guilabel:`Inventory Overview` dashboard.
 
-.. image:: delivery_three_steps/three-step-delivery-transfers.png
-   :align: center
-   :alt: Ready status for the pick operation while the packing and delivery operations are waiting
-         another operation.
-
-The status of the picking will be :guilabel:`Ready`, since the product must be picked from stock
-before it can be packed. The status of the packing order and delivery order will both be
-:guilabel:`Waiting Another Operation`, since the packing and delivery cannot happen until the
-picking is completed. The status of the delivery order will only change to :guilabel:`Ready` when
-the packing has been marked :guilabel:`Done`.
-
-The receipt can also be found in the *Inventory* application. In the :guilabel:`Overview` dashboard,
-click the :guilabel:`1 To Process` smart button in the :guilabel:`Pick` kanban card.
-
-.. image:: delivery_three_steps/three-step-kanban-pick.png
-   :align: center
-   :alt: The pick order can be seen in the Inventory Kanban view.
+Click the :guilabel:`# To Process` button, which reveals the picking order generated from the
+previously confirmed sales order.
 
 Click on the picking to process. If the product is in stock, Odoo will automatically reserve the
 product. Click :guilabel:`Validate` to mark the picking as done, and complete the transfer to the
-:guilabel:`Packing Zone`. Then the packing order will be ready. Since the documents are linked, the
-products which have been previously picked are automatically reserved on the packing order.
+:guilabel:`Packing Zone`.
 
-.. image:: delivery_three_steps/validate-three-step-pick.png
+.. image:: delivery_three_steps/delivery-three-steps-picking-order.png
```

Comment:
I'd revert back to the other screenshot.
Since the old screenshot covers more of the UI, it provides users navigational context as to where we are when we're looking at the **Delivery** smartbutton.

Reply:
Screenshot has been replaced with a new zoomed-out one, showcasing more of the navigational UI and highlighting key areas. Suggestion implemented ✅